tests/providers/dbt/cloud/operators/test_dbt_cloud.py

```diff
@@ -388,3 +389,27 @@ def test_get_text_artifact_with_step(self, mock_get_artifact, conn_id, account_i
             account_id=account_id,
             step=2,
         )
+
+
+class TestDbtCloudListJobsOperator:
+    def setup_method(self):
+        self.dag = DAG("test_dbt_cloud_list_jobs_op", start_date=DEFAULT_DATE)
+        self.mock_ti = MagicMock()
+        self.mock_context = {"ti": self.mock_ti}
+
+    @patch("airflow.providers.dbt.cloud.hooks.dbt.DbtCloudHook.list_jobs")
+    @pytest.mark.parametrize(
```

Comment:
Can you add a test where the `account_id` is not explicitly passed to the operator too?

Reply:
done with this change in the latest commit 74606ca